src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-custom-tag.js

```diff
@@ -28,7 +28,8 @@
                 upcast: (element) => {
                     return (
                         element.name === 'div' &&
-                        element.attributes['data-ezelement'] === 'eztemplate'
+                        element.attributes['data-ezelement'] === 'eztemplate' &&
+                        !element.attributes['data-eztype']
```

Comment:
This attribute is meant to disappear, so here it's safer to treat elements without it as Custom Tags.